test/browser/pages/relying-party.js

```diff
@@ -8,9 +8,8 @@ module.exports = class PlaywrightDevPage {
   }
 
   async goto() {
-    const websiteHost = process.env.WEBSITE_HOST || "http://localhost:5050";
```

Comment:
The webservice is set to run on port `5050` by default, we should stick to that in our tests as well

https://github.com/alphagov/di-ipv-cri-toy-front/blob/main/src/lib/config.js#L13